content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -66,7 +66,26 @@ primary and canary installations.
       istiod-canary-9cc9fd96f-jpc7n   1/1     Running   0          34m   canary
     {{< /text >}}
 
-1. Follow the steps [here](/docs/setup/upgrade/canary/) to test or migrate
+1. If you are using [Istio gateways](/docs/setup/additional-setup/gateway/#deploying-a-gateway), install a canary revision of the Gateway chart by setting the revision value:
+
+    {{< text bash >}}
+    $ helm install istio-ingress-canary istio/gateway \
+        --set revision=canary \
+        -n istio-ingress
+    {{< /text >}}
+
+1. Verify that you have two versions of `gateway` installed in your cluster:
```

Comment:
You mention `gateway` but I don't see that specific wording in the output. Would it be better to use `istio-ingress` or maybe `istio-ingress gateway`?